### ISSUE TITLE
[Feature] Ability to change highlight color when pressing listitem

### ISSUE THREAD

**Issue by xanderdeseyn** [Adyen/adyen-ios#1093]
![Simulator Screen Shot - iPhone 14 Pro - 2022-12-02 at 11 44 02](https://user-images.githubusercontent.com/1395284/205275370-01ee46f8-0a01-4237-8fbe-4666ae834c44.png)

The default does not fit for a dark theme.

**Comment by descorp** [comment]
Hey @xanderdeseyn 

Sorry for a long reply 😅
Are you making your own custom "dark mode" for this?

Default selection in dark mode will looks like this for v5.2.0:

![Screenshot 2023-04-19 at 13 59 11 (2)](https://user-images.githubusercontent.com/2648655/233068467-ee27e955-59b8-4671-8ebb-5fc443cd32a2.png)

**Comment by goergisn** [comment]
Closing ticket due to inactivity.

You could force drop-in into dark mode using

```
let viewController = dropIn.viewController
if #available(iOS 13.0, *) {
    viewController.overrideUserInterfaceStyle = .dark
}
present(viewController: viewController, completion: nil)
```
